### PR TITLE
worked-on-issue-63 

### LINKED ISSUE #63: [Style] - Fix layout shift
### What is your UI design change or proposal?

Have you noticed that before the forum posts are loaded and rendered, the `<footer>` is in the middle of the screen as the table is empty?
Screenshot 2022-11-23 100010
We need to change this so the table takes as much space as it would with the posts there already. You can assume that space for 30 posts will be needed initially. It does not need to be the exact size as some posts can contain more content than others.
(Solution to this may bring up another issue which will be opened once you figure out what it is)

### PR DESCRIPTION
**Pull request by dgupta0**
# Description

I think I resolved the layout shift issue of the footer. Thanks for the helpful hints. I didn't really get the `index.js` code but with the help I was able to know where to implement the change in the code.

Fixes #63

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Chore items (this includes basic clean up of files, or updates to packages)
- [ ] Updates to documentation

## Checklist

- [x] I have a descriptive title for my PR
- [x] I have linked the issue to this [PR](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)
- [x] I have run the project locally and reviewed the code changes
- [x] My changes generate no new warnings

### CODE REVIEW

**Review by AndreyDmitriev39r — APPROVED** [review]
Looks fine.
Basically hide footer until it is needed to be visible if I get your logic right

File: index.js [hunk 1]
```diff
@@ -113,6 +113,7 @@ const displayPost = (post) => {
     <td class="post-activity">${formatDateDiff(Date.now(), post.bumped_at)}</td>
   </tr>`;
   postsContainer.innerHTML += postRow;
+  document.getElementById("footer").style.display = "block";
```
Comment (Dorfieeee) [inline index.js:116]: Nice try, but I actually have some problems with this.

Now, the function `displayPost` not only adds new post to the table, but also displays the footer.
Moreover, the `displayPost` is called on each post, so it changes the style of footer `number of posts` times.

Finally, what happens when it fails to load the posts from the server? Does it mean it's not going to display the post?

Can you find a better place for this line of code?

**Comment by dgupta0** [comment]
I believe that's what you guys were suggesting, right? That's what I
understood, though 🤓

File: index.js (hunk 1 above)
Comment (dgupta0) [reply index.js:116]: Yea, that's what I imagined that it wouldn't show footer if the data doesn't load. I will give this a try again. Thanks for the feedback guys. I know I implement sheepish hacks to fix the issues

Comment (Dorfieeee) [reply index.js:116]: What if you wrap the line of code inside `displayFooter` function.
Then look between lines 25-50. Can you see a suitable place for such a function?

**Review by Dorfieeee — APPROVED** [review]
Well done.

**Pull request merged**